generators/app/templates/src/StarterKit/StarterKit.csproj

```diff
@@ -26,16 +25,17 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="AutoMapper" Version="8.1.1" />
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="6.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.5" />
```

Comment:
Kunnen we deze niet er uit laten?
Wij zijn overgestapt naar de nieuwe System.Text.Json serializer en deze behandelt out of the box al heel veel cases.
Wanneer dit onvoldoende is, voegen wij als nog de NewtonSoft package toe